src/twitter.d.ts

```diff
@@ -0,0 +1,28 @@
+import { CredentialsArgs } from './Credentials';
+import { TwitterStream } from './TwitterStream';
```

Comment:
For consistency can we switch these to `require`? Thanks!

Reply:
I don't believe we can `require` in the scope of a TypeScript definition file: https://stackoverflow.com/questions/59939513/can-a-typescript-definition-file-have-a-require-statement

Reply:
We should be able to 

```typescript
const { CredentialsArgs } = require('./Credentials');
```

But I don't feel strongly and am happy to merge either. If it needs to be changed, I'm sure an issue will be raised :P

Reply:
Huh 🤔 VS Code was definitely giving me some trouble when I tried to swap it to a `require` import instead. But yeah, if it causes an issue I'm sure it will be brought up 😛 Thanks for merging this in!